src/keys/drivers/YubiKeyInterfacePCSC.cpp

```diff
@@ -530,109 +529,98 @@ YubiKeyInterfacePCSC* YubiKeyInterfacePCSC::instance()
     return m_instance;
 }
 
-void YubiKeyInterfacePCSC::findValidKeys()
+bool YubiKeyInterfacePCSC::findValidKeys()
 {
     m_error.clear();
     if (!isInitialized()) {
-        return;
+        return false;
     }
-
```

Comment:
The block changes below are because of indent changes

Reply:
GitHub has an option to hide those white-space-only changes.